event/DetailsQuery.ts

```diff
@@ -18,7 +18,10 @@ export const useEventDetailsQuery = (
   id: EventID,
   loadEvent: (id: EventID) => Promise<EventDetailsLoadingResult>
 ) => {
-  return useQuery(eventDetailsQueryKey(id), async () => await loadEvent(id))
+  return useQuery({
+    queryKey: eventDetailsQueryKey(id),
+    queryFn: async () => await loadEvent(id)
+  })
```

Comment:
Move `useLoadEventDetails` to this file and inline this `useQuery`. I can't see any other component wanting to use the raw query for the event details, so it's just better to consolidate this into a single hook.